### PR TITLE
fix: Delete should only be triggered on finding DELETE symbol

### PR DESCRIPTION
**Pull request by ntucker**
### Motivation

Deletes have special behavior and should never trigger when something else is awry.

### CODE REVIEW

**Review by ljharb — APPROVED** [review]
are there any tests that aren't just snapshots? the tests in this PR don't make it clear in any way what the problem was

**Comment by ntucker** [comment]
Adding some console warnings and a test to capture this

File: packages/normalizr/src/schemas/__tests__/__snapshots__/Array.test.js.snap [hunk 1]
```diff
@@ -969,6 +969,31 @@ Array [
     ],
     undefined,
   ],
+  Array [
```
Comment (ntucker) [inline packages/normalizr/src/schemas/__tests__/__snapshots__/Array.test.js.snap:972]: not the best test....as the snapshot needs to be updated based on the extra calls to this.getSchemaAttribute() to format the console warning

**Comment by ntucker** [comment]
@christiancho you might like this one ;)

**Pull request merged**